src/x/headers/headers.go

```diff
@@ -91,6 +91,10 @@ const (
 	// the number of docs returned by each storage node.
 	LimitMaxDocsHeader = M3HeaderPrefix + "Limit-Max-Docs"
 
+	// LimitMaxRangeHeader is the M3 limit range header that limits
+	// the time range returned by each storage node.
+	LimitMaxRangeHeader = M3HeaderPrefix + "Limit-Max-Range"
```

Comment:
nit: if this is aggregate query specific, should scope the header name to indicate this

Reply:
It's applicable to both 👍

Reply:
(Updated integration tests to match)